packages/block-library/src/gallery/edit.js

```diff
@@ -104,23 +104,38 @@ function GalleryEdit( props ) {
 		getSettings,
 		preferredStyle,
 		innerBlockImages,
-		wasBlockJustInserted,
+		blockWasJustInserted,
+		multiVGallerySelection,
```

Comment:
```suggestion
		multiGallerySelection,
```

Typo?

Reply:
Fixed in [d013d08](https://github.com/WordPress/gutenberg/pull/57378/commits/d013d08e6959b543fa66b0e5032879cec8419085).